ItalianTranslation_PlanetCoaster.xml

```diff
@@ -4521,124 +4521,124 @@
-		<entry key="FanfarePackCompletion_ELLY_2" translation="You collected all the Silver stars for 'Miss Elly's Round Up'."/>
-		<entry key="FanfarePackCompletion_ELLY_3" translation="You collected all the Gold stars for 'Miss Elly's Round Up'."/>
-		<entry key="FanfarePackCompletion_ELLY_All" translation="You have completed every objective in 'Miss Elly's Round Up'!"/>
-		<entry key="FrontEndMenu_ScenarioDetails_DINE" translation="Miss Elly's Diner has some great facilities, including a big wooden coaster for a spot of excitement! But its remote location means it doesn’t see many visitors, and its potential isn’t being realised. Can you build on this location and draw in the crowds?"/>
-		<entry key="FrontEndMenu_ScenarioDetails_MARS" translation="They filmed some pretty famous movies out here in the desert. It kind of looks like Mars, don’t it. The place isn’t much in demand any more – it’s all CGI these days – but the company doesn’t want to let it go. They want you to turn it into a theme park instead!"/>
-		<entry key="FrontEndMenu_ScenarioDetails_MINE" translation="Some poor fella bought this mine from a prospector, and took out some big loans to pay for it. Turns out it was a scam. The varmint had salted the mine! Now he’s trying to turn it into a tourist attraction with some fireworks he was given and Old West-style streets, but it’s not doing as well as he hoped and he needs your help."/>
-		<entry key="FrontEndMenu_ScenarioName_DINE" translation="Miss Elly's Diner"/>
-		<entry key="FrontEndMenu_ScenarioName_MARS" translation="Starship Hangar"/>
-		<entry key="FrontEndMenu_ScenarioName_MINE" translation="Goldmine Tower"/>
-		<entry key="FrontEndMenu_SetDescription_ELLY" translation="You gotta make a big splash to succeed here. Can you tame the wild frontier?"/>
-		<entry key="FrontEndMenu_SetName_ELLY" translation="Miss Elly's Round Up"/>
+		<entry key="FanfareCareerSetUnlocked_ELLY" translation="Miss Elly sta cercando un aiuto per il ranch, ma non è il bestiame che sta radunando, sono i visitatori!"/>
+		<entry key="FanfarePackCompletion_ELLY_1" translation="Hai collezionato tutte le stelle di Bronzo per 'Il raduno di Miss Elly'."/>
+		<entry key="FanfarePackCompletion_ELLY_2" translation="Hai collezionato tutte le stelle d'Argento per 'Il raduno di Miss Elly'."/>
+		<entry key="FanfarePackCompletion_ELLY_3" translation="Hai collezionato tutte le stelle d'Oro per 'Il raduno di Miss Elly'."/>
+		<entry key="FanfarePackCompletion_ELLY_All" translation="Hai completato ogni obiettivo per 'll raduno di Miss Elly'!"/>
+		<entry key="FrontEndMenu_ScenarioDetails_DINE" translation="Il ristorante di Miss Elly ha delle strutture eccezionali, compreso un grande coaster di legno per un pò di eccitazione! Ma la posizione remota signica non avere molti visitatori e il suo potenziale non viene realizzato. Riesci a costruire in questo posto e attirarci le folle?"/>
```

Comment:
Non mi piace molto questo, soprattutto "Riesci a costruire in questo posto e attirarci le folle?"
Potresti pensare a qualcosa di meglio? Non so anche mettere:
 "Riuscirai a costruire in questa zona e attirare le folle?" - Qualcosa di più semplice, non so se mi spiego

Reply:
Ok, vado di traduzione libera:
Ma visto che si trova in una località remota non ha molti visitatori e il suo potenziale è inespresso ---- poi vado con la tua frase